README.md

```diff
@@ -1,27 +1,102 @@
+```mermaid
+flowchart TD
+    A("edi-energy-mirror:
+    raw documents (PDF, docx)") -->|kohlrAHBi🥬| B("machine-readable-
+    anwendungshandbücher")
+    A -->|kohlrAHBi🥬| C("Azure Blob Storage")
+    C <--> D("AHBesser
+    (AHB tables)")
+```
 
-## Code scaffolding
+### 📂 Project Structure
+
+```
+. 
+├── azure-mock/
+    ├── data/                     # contains AHB data stored in machine-readable_anwendungshandbuecher repository
+    └── upload-documents.ts       # script to upload data/ directory to azure blob storage
+├── src/
+    ├── app/
+        ├── core/
+            └── api/              # API config files
+        ├── environments/         # config files for dev/stage/prod
+        ├── features/             # AHB and landingpage related views and components
+        └── shared/               # global components (header, footer, logo, etc.)
+    ├── assets/                   # logo, favicon, etc.
+    ├── server/
+        ├── controller/           # contains code to handle incoming http requests concerning AHB and FormatVersionen 
+        ├── infrastructure/       # contains code to manage routing of API endpoints and interact with azure blob storage
+        └── repository/           # contains CRUD operations to register AHB/FormatVersionen related routers
+    ├── index.html                # entry point for the angular web application
+    ├── main.ts                   # bootstraps the angular web application
+    ├── server.ts                 # sets up backend server
+    └── styles.scss               # imports Tailwind base styles, component styles and utility classes
+└── ⚙️ <several config files>
+```
+
+## ⚙️ Setup
+
+Make sure you have the latest version of [node.js](https://nodejs.org/en) installed (for instance via the [node version manager](https://github.com/nvm-sh/nvm) `nvm`).
+
+Download and install [Angular CLI](https://v17.angular.io/cli) using the `npm` package manager (comes with node.js):
+
+```bash
+$ npm install -g @angular/cli
+```
+
+[**Windows**] Add node.js to your PATH environment variable:
+
+- run `$ npm config get prefix` to retrieve the directory where npm will install global packages (e.g. `C:\Program Files\nodejs`)
+- open "Edit the system environment variables" and navigate to "Environment Variables" -> "System Variables" -> "Path"
+- edit "Path" and add the node.js directory path
+- restart your PC and check if Angular CLI has been installed successfully by running `$ ng --version`
+
+### Starting the app using Angular CLI
+
+To start a dev server, run
+
+```bash
+$ ng serve
+```
+
+and navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+
+### Starting the app via Docker 🐋
+
+While having [Docker Desktop](https://www.docker.com/products/docker-desktop/) up and running, start the docker container using
+
+```bash
+$ docker compose up -d
```

Comment:
I guess this would not work if you have an initial start, because you have no docker image of the AHBesser application.

I would recommend to add the `--build` flag to it.

```suggestion
$ docker compose up -d --build
```

Reply:
This will create the image and therefor takes all the latest code changes into the image.